test/v1/listUkSavingsAccounts/def1/model/response/Def1_ListUkSavingsAccountsResponseSpec.scala

```diff
@@ -46,7 +46,49 @@ class Def1_ListUkSavingsAccountsResponseSpec extends UnitSpec {
     """.stripMargin
   )
 
-  val ukSavingsAccountsFromMTD = Json.parse(
+  private val ukSavingsAccountsIfsJs = Json.parse(
+    """
+      |{
+      |  "bbsi": [
+      |      {
+      |        "incomeSourceId": "000000000000001",
+      |        "incomeSourceName": "Bank Account 1",
+      |        "identifier": "AA111111A",
+      |        "incomeSourceType": "interest-from-uk-banks"
```

Comment:
Can you please update the _ukSavingsAccountsIfsJs_  and _ukSavingsAccountsNotBbsiIfsJs_ to include the correct fields _startDate_ and _endDate_ instead of _identifier_ and _incomeSourceType_